schlib/comparelibs.py

```diff
@@ -144,23 +146,57 @@ def KLCCheck(lib, component):
 
             if args.verbose:
                 printer.yellow("Changed symbol '{lib}:{name}'".format(lib=lib_name, name=cmp))
+            if args.design_breaking_changes:
+                pins_moved = 0
+                nc_pins_moved = 0
+                pins_missing = 0
+                nc_pins_missing = 0
+                for pin_old in old_cmp[cmp].pins:
+                    pin_new = new_cmp[cmp].getPinByNumber(pin_old['num'])
+                    if pin_new is None:
+                        if pin_old['electrical_type'] == 'N' and pin_new['electrical_type'] == 'N':
+                            nc_pins_missing +=1
+                        else:
+                            pins_missing += 1
+                        continue
+
+                    if pin_old['posx'] != pin_new['posx']:
+                        if pin_old['electrical_type'] == 'N' and pin_new['electrical_type'] == 'N':
+                            nc_pins_moved +=1
+                        else:
+                            pins_moved += 1
+                    elif pin_old['posy'] != pin_new['posy']:
+                        if pin_old['electrical_type'] == 'N' and pin_new['electrical_type'] == 'N':
+                            nc_pins_moved +=1
+                        else:
+                            pins_moved += 1
+
+                if pins_moved > 0 or pins_missing > 0:
+                    design_breaking_changes += 1
+                    printer.light_purple("Pins have been moved, renumbered or removed in symbol '{lib}:{name}'".format(lib=lib_name, name=cmp))
+                elif nc_pins_moved > 0 or nc_pins_missing > 0:
+                    design_breaking_changes += 1
+                    printer.purple("Normal pins ok but NC pins have been moved, renumbered or removed in symbol '{lib}:{name}'".format(lib=lib_name, name=cmp))
```

Comment:
Could remove the "Normal pins ok but " part to keep this succinct.

Reply:
Maybe the wording can be improved but i really want to include the fact that normal pins have not been touched. This is to make it absolutely clear that this symbol should not break 90% of the designs that use it (or maybe it is even closer to 100% to be honest.)

Reply:
An option would be to make it clear in the other output that these are the important pins that have been moved. Not sure how to word this.